packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -388,29 +388,29 @@ export interface IMergeTreeOptions {
     /**
      * Whether to enable the length calculations implemented in
      * https://github.com/microsoft/FluidFramework/pull/11678
-     * 
+     *
```

Comment:
I tried to remove these spacing changes, but it removed the rest of my changes in this file as well. I then tried removing all of the changes and adding back just the ones I needed for this fix, but the spacing changes reappeared in the file. Not sure why they keep coming back, but I will continue trying to revert them.

Reply:
Auto-formatting removing trailing spaces is fine IMO.